### PR TITLE
Started experimenting with celery for distributed task processing

### PR DESCRIPTION
**Pull request by gunthercox**
This pull request represents an investigation into using [Celery](http://www.celeryproject.org/) and [RabbitMQ](https://www.rabbitmq.com/) for distributed task processing. The goal for these changes is to provide a way to add a number of distributed worker servers  to help the robot process information. This will be extremely important in the future as the robot begins to need more computing power for processing image and audio input.

### CODE REVIEW

File: apt-requirements.txt [hunk 1]
```diff
@@ -11,5 +11,9 @@ sudo apt-get install jackd2
 
 sudo apt-get install python-pip pyaudio
 sudo apt-get install bison libasound2-dev swig python-dev mplayer
+
 # required only if the system is not x86-based Windows/Linux/OS X
 sudo apt-get install flac
+
+# RabbitMQ
+sudo apt-get install rabbitmq-server
```
Comment (vkosuri) [inline apt-requirements.txt]: Master, How about using ansible https://www.ansible.com/ For example

```
ansible-playbook -i ansible/apt-requirements.yml -c local --ask-sudo-pass
```

Comment (gunthercox) [inline apt-requirements.txt]: Ansible looks interesting. I'll definitely have to look into it a bit more as their website is a bit vague.

Comment (vkosuri) [inline apt-requirements.txt]: can i try?

Comment (gunthercox) [inline apt-requirements.txt]: Sure, feel free to.

Comment (vkosuri) [inline apt-requirements.txt]: Apologies, Ansible works wellwith python 2.x, i tried on pyton3 it produces different problems,  http://docs.ansible.com/ansible/intro_installation.html#control-machine-requirements

**Pull request closed without merging**